paper/paper.md

```diff
@@ -41,18 +41,18 @@ TO DO
 # HyP3 Description
 
 ## Development History
-HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2017 by a group of undergraduate University of Alaska Fairbanks students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows that ran on-premise and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
+HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2016 by a group of undergraduate students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows running on-premises and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
 
-By 2019 it was clear that re-designing HyP3 to take advantage of evolving technologies (namely the new AWS Batch service, infrastructure-as-code, and containerized workflows) would greatly improve its utility. Thus, in 2019 HyP3 was re-designed to exist entirely within the AWS cloud, with application-specific processing workflows encapsulated within Docker containers. The re-design was completed in 2020, and on-demand SAR image processing services that utilized HyP3 were made available to the public for free via an API, Python SDK, and ASF's Data Search Vertex website.
+By 2019 it was clear that re-designing HyP3 to take advantage of evolving technologies (namely the new AWS Batch service, infrastructure-as-code, and containerized workflows) would greatly improve its utility. Thus, in 2019 HyP3 was re-designed to run entirely within the AWS cloud, with application-specific processing workflows encapsulated within Docker containers. The re-design was completed in 2020, and on-demand SAR image processing services that utilized HyP3 were made available to the public at no cost via an API, Python SDK, and ASF's Data Search Vertex website.
```

Comment:
The "for free" vs. "at no cost" is one we've struggled with a lot in our documentation. Maybe "made publicly available at no cost to users via an API"... might work?

Reply:
Yeah, the term "free" is often used in the context of "free software", meaning software provided under a libre license. Of course, that [also applies to HyP3](https://github.com/ASFHyP3/hyp3/blob/develop/LICENSE), but I assume that in this context we want to specifically highlight the no-cost aspect of HyP3. So I think "at no cost" is generally more explicit and a bit more readable than "free".